EIPS/eip-draft-extbalance.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2604
+title: EXTBALANCE rename and conditional pricing
+author: wjmelements.eth <wjmelements@gmail.com> (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2604-conditional-extbalance-repricing/
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-16
+requires: 1884
+---
+
+## Simple Summary
+Renames `BALANCE` to `EXTBALANCE` and reduces `EXTBALANCE` cost when querying the current contract.
+
+
+## Abstract
+This EIP renames `BALANCE` to `EXTBALANCE`.
+The gas cost of `EXTBALANCE` when the parameter matches the current address is reduced to a value slightly higher than the cost of `SELFBALANCE`.
+
+
+## Motivation
+As the primary purpose of `EXTBALANCE` following EIP1884 is to query external accounts, the name is updated accordingly.
```

Comment:
```suggestion
As the primary purpose of `EXTBALANCE` following [EIP-1884](./eip-1884.md) is to query external accounts, the name is updated accordingly.
```

Reply:
All EIP's should be referred to as EIP-### and at *least* the first reference to any given EIP in an EIP should include a relative path link to it.